tests/advanced.test.ts

```diff
@@ -231,23 +231,36 @@ describe(`Advanced`, () => {
     });
 
     it(`supports strings that act like booleans if not bound to a value`, async () => {
-        const commandGenerator = () => {
-            class CommandA extends Command {
-                @Command.String(`--break`, { booleanIfNotBound: true })
-                enableDebugger: boolean | string = false;
+        class CommandA extends Command {
+            @Command.String(`--break`, { booleanIfNotBound: true })
+            enableDebugger: boolean | string = false;
 
-                async execute() {
-                    log(this, [`enableDebugger`]);
-                }
+            async execute() {
+                log(this, [`enableDebugger`]);
             }
-            return [
-                CommandA,
-            ];
-        };
+        }
+
+        class InvertedCommandA extends Command {
+            @Command.String(`--break`, { booleanIfNotBound: true })
+            enableDebugger: boolean | string = true;
+
+            async execute() {
+                log(this, [`enableDebugger`]);
+            }
+        }
+
+        let cli = Cli.from([CommandA])
+
+        expect(cli.process([])).to.contain({enableDebugger: false});
+        expect(cli.process([`--break`])).to.contain({enableDebugger: true});
+        expect(cli.process([`--no-break`])).to.contain({enableDebugger: false});
```

Comment:
Might also be a good idea to test that `cli.process(['--no-break=1234'])` throws, since it wouldn't make sense to bind it.